classes/PodsMigrate.php

```diff
@@ -636,10 +641,15 @@ public function build_xml( $data = null ) {
 			return false;
 		}
 
-		$head  = '<' . '?' . 'xml version="1.0" encoding="utf-8" ' . '?' . '>' . "\r\n<items count=\"" . count( $this->data['items'] ) . "\">\r\n";
+		$items = $this->data['items'];
```

Comment:
This is the second time this logic appears - it should be in its own private method: